_rules/meta-refresh-no-delay-bc659a.md

```diff
@@ -85,7 +85,7 @@ This rule is designed specifically for [2.2.1 Timing Adjustable][sc221], which c
 
 #### Passed Example 1
 
-Redirects immediately.
+This page redirects to a new page immediately.
```

Comment:
There's a bit of  a general discrepancy in the descriptions that sometimes refer to "this page" and sometimes to "this meta element". Given that the rule tangets the `meta` element, I guess we should use "this meta element" or "the meta element on this page".

This is however fairly minor, and descriptions are good as they are.